contrib/docker/libvirt/start.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -o errexit
+
+modprobe kvm
```

Comment:
This seems to be problematic inside of container. Did you tested this with running `modprobe -r kvm` on host before starting this dock?